project/plugins.sbt

```diff
@@ -19,3 +19,5 @@ addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.6.0")
 addSbtPlugin("com.typesafe.sbt" % "sbt-multi-jvm" % "0.3.8")
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "5.2.1")
 addSbtPlugin("com.typesafe" % "sbt-mima-plugin" % "0.1.13")
+
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.8.2")
```

Comment:
I had included the `sbt-dependency-graph` plugin on `master` but not in `1.3.x`. Proved useful.

Reply:
I don't think we should include it here. This is better added on ~/.sbt/0.13/plugins/plugins.sbt